AmazonChimeSDK/AmazonChimeSDKTests/device/DeviceControllerTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DeviceControllerTests.swift
+//  AmazonChimeSDK
+//
+//  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//  SPDX-License-Identifier: Apache-2.0
+//
+
+@testable import AmazonChimeSDK
+import AVFoundation
+import XCTest
+
+class DeviceControllerTests: XCTestCase {
+    func testListDevicesReturnsSpeakerWithCorrectType() {
+        let audioSession = MockAudioSession()
+        let videoClient = MockVideoClientController()
```

Comment:
Can you use Mockingbird auto generated mock class instead of creating one on your own?

It should be as simple as `let videoClient = mock(VideoClientController.self)`, examples: https://github.com/aws/amazon-chime-sdk-ios/commit/a555de4f5bc5fe9743d21ff764d9f2819c6dd060